### DISCUSSION TITLE
OpenFreeMap ❤️ MapLibre

### DISCUSSION THREAD

**Discussion by hyperknot** [maplibre/maplibre-gl-js#4736] [Show and tell]
Hi Maplibre!

After 9 years of running my own OpenStreetMap tile server infra for [MapHub](https://maphub.net/), I've open-sourced it and launched OpenFreeMap. 

OpenFreeMap provides 3 things:
- A public instance for full planet OSM vector tiles, updated weekly. 
- 100% open-source.
- Weekly processed full planet downloads both in Btrfs and MBTiles formats.

All our example code and docs use MapLibre.

Using the public instance is completely free: there are no limits on the number of map views or requests. There’s no registration, no user database, no API keys, and no cookies. I aim to cover the running costs of the public instance through donations.

Everything is open-source, including the full production setup — there’s no ‘open-core’ model here. Check out our [GitHub](https://github.com/hyperknot/openfreemap). The map data comes from OpenStreetMap.

Looking forwards for your feedback.

**Comment by HarelM** [comment]
Those are great news!! Thanks for sharing! 
Do you think we should update the examples to use these tiles? It might create a cost issue...?
Also it might be interesting to publish it as part of our monthly newsletter. 
Cc: @wipfli.

Thanks again!

**Reply by wipfli** [reply]
Thanks for sharing, @hyperknot! I think it is best to start by adding this to the list of map service providers in the awesome-maplibre repo.

@HarelM regarding the MapLibre GL JS docs examples: we are in the middle of drafting a policy in the Governing Board and I would like to finish it before we change anything on the current setup.

**Reply by HarelM** [reply]
Fair enough.
Please take into consideration the fact that there might be "editing wars" from providers who gain benefit from showcasing their tiles in the examples pages and I would like to avoid it as much as possible by making the examples as neutral as possible, besides maybe some specific pages to showcase certain providers.

**Comment by hyperknot** [comment]
From my side, I'm more than happy if you use OpenFreeMap in the official
examples.

I think it'll help with the accessibility for new devs, as they don't have
to sign up to accounts and API keys.

Of course I guess it's a board level decision, and you might have sponsors,
etc., so all I can do is offer the service.
